src/parser/node/mod.rs

```diff
@@ -72,6 +73,7 @@ fn base_node(input: Span) -> IResult<Located<Node>> {
         }),
         lookahead(pair(tag("if"), multispace1), cond::cond),
         lookahead(tag("fn"), fn_def::fn_def),
+        lookahead(primitive, call::call),
```

Comment:
you made me wonder. What happens with the parser if you try to parse `(print)(10)`?